python/pyspark/mllib/feature.py

```diff
@@ -271,29 +271,74 @@ def transform(self, vector):
         """
         return JavaVectorTransformer.transform(self, vector)
 
+class ChiSqSelectorType:
+    """
+    This class defines the selector types of Chi Square Selector.
+    """
+    KBest, Percentile, FPR = range(3)
```

Comment:
Hi @srowen , there are many ways to express enums in Python 2, this is the most frequently used and simplest way.